containers/podman
# Add restart-sec option to systemd generate command

#### What this PR does / why we need it:
This PR add new parameter called `restart-sec` to `podman generate systemd` command. This parameter is used to set `RestartSec` parameter in `[Service]` section of the pod unit in systemd. The `RestartSec` configures the time to sleep before restarting a service. Some users may need to set this variable to make sure the service is controlled by default mechanism.



#### How to verify it
When running following command:
```bash
podman generate systemd --restart-sec 15 pod_name
```
You should see in the pod unit [Service] section `RestartSec=15`.



#### Which issue(s) this PR fixes:



#### Special notes for your reviewer:
